public/app/features/logs/components/LogRow.tsx

```diff
@@ -108,6 +109,14 @@ class UnThemedLogRow extends PureComponent<Props, State> {
     if (!this.props.enableLogDetails) {
       return;
     }
+
+    if (!this.state.showDetails) {
+      reportInteraction('grafana_explore_logs_log_details_opened', {
```

Comment:
I know this interaction is based on the Excel document. But what do you think about reporting the interaction everytime we toggle the details and move the `showDetails` state into the report's properties?

Also, what do you think about adding the LogRow's `uid`? With that we could quite easy solve the part in the Excel document regarding `Duration of how long do they spend here`.

However, those are just ideas and I am also happy without the suggested changes.

Reply:
great idea! So the interaction would be `grafana_explore_logs_log_details_clicked` and the properties would include `type: open/close` and `logRowUid` with `row.uid`.

Reply:
i like this suggestion, i'll change it now!